tests/vector_tools/interpolate.cc

```diff
@@ -0,0 +1,282 @@
+#include <deal.II/base/utilities.h>
+
+#include <deal.II/distributed/tria.h>
```

Comment:
We typically place a description of what the test does right after the copyright header and before the include files start. This way, one can figure out what a test does without having to read through a couple hundred lines of code :-)

Reply:
Description added.